src/app/modules/customer-management/components/customer-info/components/create-customer/create-customer.html

```diff
@@ -23,6 +24,7 @@
         rows="3"
         formControlName="description"
         placeholder="Customer description"
+        (input)="onSearchChanges($event.target.value)"
```

Comment:
I have a doubt. Why is the function called `onSearchChanges`? I remember the purpose is to check when the value Input field has changed. Maybe we need better name like `onInputChange` or something similar.

Reply:
Yes, the purpose is to check when the `Input` value or the `textArea` has changed.
I'm going to rename `onSearchChanges`